bundle/jsky.app.ot/src/main/java/jsky/app/ot/gemini/inst/SciAreaFeature.java

```diff
@@ -248,6 +250,11 @@ public void reinit(TpeImageWidget iw, TpeImageInfo tii) {
                 _ghost2Feat = new SciAreaPlotFeature(GhostScienceAreaGeometry$.MODULE$);
             }
             _feat = _ghost2Feat;
+        } else if (inst instanceof Igrins2) {
+            if (_igrins2Feat== null) {
+                _igrins2Feat = new SciAreaPlotFeature(GhostScienceAreaGeometry$.MODULE$);
```

Comment:
Is using GHOST intentional?  Is the science area feature required for the OT Browser?

Reply:
Ups this belongs to another PR 😄